File: C/Ex35.c
#include <stdio.h>
#include <stdlib.h>
int main(int argc, char *argv[]){
  int numeros[20];
  int multiplicador;
  
  printf("Digite os 10 numeros: ");
  for(int i=0; i<20; i++){
      scanf("%i", &numeros[i]);
  printf("Deve multiplicar por quanto? \n");
  scanf("%i", &multiplicador);
      numeros[i] = numeros[i] * multiplicador;
  }
 
  printf("Os numeros digitados multiplicados pelos respectivos valores:  ");
  for(int i=0; i<20; i++){
      printf("%i \n", numeros[i]);
  }
 
  return 0;
}

File: Python/Ex35.py
numeros = []
for i in range(0, 20, 1):
    x = int(input('Digite um numero: '))
    if(i == 0):
        constante = int(input('Digite a constante multiplicativa: '))
    x= x* constante
    numeros.append(x) 
    
print('Os numeros digitados multiplicados pela constante: ')

for i in range(0, 20, 1):
    print(numeros[i])